src/main/webapp/scheduled-interviews.jsp

```diff
@@ -23,6 +23,16 @@
             <li class="list-group-item">Interviewee: ${scheduledInterview.getInterviewee()}</li>
             <li class="list-group-item">Interviewer: ${scheduledInterview.getInterviewer()}</li>
           </ul>
+          <c:choose>
+            <c:when test= "${scheduledInterview.getHasPassed()}">
+              <a href="feedback.html?interview=${scheduledInterview.getId()}&role=${scheduledInterview.getRole()}" style="text-align:center">
+                <button class="btn btn-primary" type="button">Submit Feedback</button>
```

Comment:
Can we disable this button when the user isn't allowed to access the form?

Reply:
When the user isn't allowed to access it, the form doesn't appear at all. Instead there's a "you don't have access yet" type message. Look in the jsp file for the specifics.